boards/atsame54_xpro/Cargo.toml

```diff
@@ -0,0 +1,67 @@
+[package]
+name = "atsame54_xpro"
+version = "0.1.1"
+authors = [
+    "Karsten Große <karsten.grosse@sympatron.de>"
+]
+description = "Board Support crate for the SAM E54 Xplained Pro Evaluation Kit"
+keywords = ["no-std", "arm", "cortex-m", "embedded-hal"]
+license = "MIT OR Apache-2.0"
+edition = "2018"
+repository = "https://github.com/atsamd-rs/atsamd"
+readme = "README.md"
+
+[dependencies]
+cortex-m = "0.6"
+embedded-hal = "0.2.3"
+smart-leds = "0.3"
+ws2812-spi = "0.2.0"
```

Comment:
Is there a ws2812 on this board? I don't see it used in the lib.rs.

Reply:
No, I will remove it.